### PR TITLE
ported to net standard 1.4

### LINKED ISSUE #15: UWP/.NET Portable support
It would be awesome if this library converted into a .NET Portable library (or .NET Standard 1.4 (not 2.0 because 1.4 is supported by most UWP versions, also I recommend .NET Portable)) to support more platforms like UWP

### PR DESCRIPTION
**Pull request by janniksam**
Resolves #15

### CODE REVIEW

File: Akinator.Api.Net/Utils/StringExtensions.cs [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+
         public static string UrlEncode(this string input)
         {
             if (input == null)
             {
                 throw new ArgumentNullException(nameof(input));
             }
 
-            var temp = HttpUtility.UrlEncode(input, Encoding.UTF8).ToCharArray();
+            var temp = UrlEncode(input, Encoding.UTF8).ToCharArray();
```
Comment (ahmed605) [inline Akinator.Api.Net/Utils/StringExtensions.cs]: @janniksam why did you have removed `HttpUtility.UrlEncode` and used another not official code?

Comment (janniksam) [reply Akinator.Api.Net/Utils/StringExtensions.cs]: HttpUtitlity wasn't there back in 1.4. And I did not find any lightweight nuget package to use.

Comment (ahmed605) [reply Akinator.Api.Net/Utils/StringExtensions.cs]: @janniksam 
IMG_20200411_110501

Comment (ahmed605) [reply Akinator.Api.Net/Utils/StringExtensions.cs]: So you can use WebUtility instead of HttpUtility

Comment (janniksam) [reply Akinator.Api.Net/Utils/StringExtensions.cs]: thank you, good point. Done.

**Review by ahmed605 — APPROVED** [review]

**Comment by ahmed605** [comment]
Looks good 👍

**Comment by ahmed605** [comment]
@janniksam don't forget to change the repo description

**Pull request merged**